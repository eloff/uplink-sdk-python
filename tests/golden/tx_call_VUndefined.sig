r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 96260078696741757037616714573465413075692189367085355034246138594266431884444
