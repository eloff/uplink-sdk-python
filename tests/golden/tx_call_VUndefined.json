{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "tag": "VUndefined"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE05NjI2MDA3ODY5Njc0MTc1NzAzNzYxNjcxNDU3MzQ2NTQxMzA3NTY5MjE4OTM2NzA4NTM1NTAzNDI0NjEzODU5NDI2NjQzMTg4NDQ0NA=="
}